documentation/components/tutorial-flow-grid.asciidoc

```diff
@@ -876,15 +891,18 @@ grid.addSortListener(event -> {
 });
 ----
 
-== Styling
+== Styling the Grid
+
+// I found the original content here difficult to understand and have interpreted as best I can
+
+Styling the `Grid` component (or any Flow component) is straightforward, but requires some Web Component and shadow-DOM knowledge. Styling depends on the components position in the DOM:
```

Comment:
```suggestion
Styling the `Grid` component (or any Flow component) requires some Web Component and shadow-DOM knowledge. Styling depends on the components position in the DOM:
```

I generally don't like that we try to make things easier by saying they are easy. Let's leave this kind of expressions away.

Reply:
I agree.